core/pkg/resourcesprioritization/prioritizationhandler.go

```diff
@@ -63,7 +111,7 @@ func (handler *ResourcesPrioritizationHandler) PrioritizeResources(sessionObj *c
 		prioritizedResource.SetSeverity(prioritizedResource.CalculateSeverity())
 		prioritizedResource.SetScore(prioritizedResource.CalculateScore())
 
-		if handler.skipZeroScores && prioritizedResource.GetScore() == 0 {
+		if skipZeroScore && prioritizedResource.GetScore() == 0 {
```

Comment:
Can you please elaborate on this logic?

Reply:
I added a flag to skip resources with score of zero (mainly used during testing of the feature)
If you believe it is not needed (so that we never create an object in case the score is 0) I can remove it

Reply:
Yes, I think you should remove it